src/main/java/yuki/Yuki.java

```diff
@@ -0,0 +1,229 @@
+package yuki;
+
+import yuki.exceptions.YukiExceptions;
+import yuki.task.Deadline;
+import yuki.task.Event;
+import yuki.task.Task;
+import yuki.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileWriter;
+
+public class Yuki {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    private static String[] data;
+    private static String description;
+
+    private static final String PATH_TO_FILE = "data/tasks.txt";
+
+    public static void readFile() throws FileNotFoundException {
+        File f = new File(PATH_TO_FILE);
+        Scanner s = new Scanner(f);
+        String line;
+        while (s.hasNext()) {
+            line = s.nextLine();
+            if (line.charAt(7) == 'T') {
```

Comment:
Try not to use magic literals e.g. magic numbers and magic string, consider creating a constant and use the constant in your code instead.